### PR TITLE
Readjust the margin

### PR DESCRIPTION
**Pull request by trevorjonesd2l**
Fix the margin so the grid grids again. Should fix this issue:

image

### CODE REVIEW

File: d2l-course-tile.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
 				</div>
 				<div class="course-text d2l-heading-4">{{_organization.properties.name}}</div>
 				<!-- Uncomment this line to get a temporary choose image button -->
-				<!-- <button on-click="_launchCourseTileImageSelector">Choose Image</button> -->
+				<button on-click="_launchCourseTileImageSelector">Choose Image</button>
```
Comment (trevorjonesd2l) [inline d2l-course-tile.html]: - [ ] TODO: Remove after tests and stuff

**Comment by jstefaniuk-d2l** [comment]
Code LGTM (I'd run this by Bob as well)

**Comment by ryantmer** [comment]
Is this more durable than the previous solution?

**Comment by jstefaniuk-d2l** [comment]
@ryantmer Not actually sure why this broke, it probably has something to do with a change here: https://github.com/Brightspace/d2l-my-courses-ui/pull/129 (since that's the only other PR)

Since this grid is pretty much the same as the course tiles (just simplified and resized), I'm not sure we need to worry that much about robustness (Once we hook it up to the actual ui, anything breaking it will be caught with testing)

**Comment by ryantmer** [comment]
Hm, maybe - thought it was in master before merging that, granted 😮

**Comment by ryantmer** [comment]
Poked around with this, seems to work a-okay.

**Pull request merged**